ethrpc.go

```diff
@@ -36,13 +36,14 @@ type ethRequest struct {
 
 // EthRPC - Ethereum rpc client
 type EthRPC struct {
-	url   string
-	Debug bool
+	url    string
+	client http.Client
+	Debug  bool
 }
 
 // NewEthRPC create new rpc client with given url
-func NewEthRPC(url string) *EthRPC {
-	return &EthRPC{url: url}
+func NewEthRPC(url string, client http.Client) *EthRPC {
```

Comment:
client must be optional.

Reply:
```
func NewEthRPC(url string, client *httpClient) *EthRPC
```
ok?

Reply:
Thank you for your advice.
I'll fix my PR later.